test_runner/src/main/kotlin/ftl/run/platform/android/GetInstrumentationShardChunks.kt

```diff
@@ -0,0 +1,54 @@
+package ftl.run.platform.android
+
+import com.linkedin.dex.parser.DexParser
+import ftl.args.AndroidArgs
+import ftl.args.ArgsHelper
+import ftl.args.ShardChunks
+import ftl.config.FtlConstants
+import ftl.filter.TestFilter
+import ftl.filter.TestFilters
+import ftl.run.model.InstrumentationTestApk
+import ftl.util.FlankTestMethod
+import ftl.util.downloadIfNeeded
+import java.io.File
+
+fun getInstrumentationShardChunks(
+    args: AndroidArgs,
+    testApks: List<InstrumentationTestApk>
+): Map<InstrumentationTestApk, ShardChunks> =
+    getFlankTestMethods(
+        testApks = testApks.download(),
+        testFilter = TestFilters.fromTestTargets(args.testTargets)
+    ).mapValues { (_, testMethods: List<FlankTestMethod>) ->
+        when {
+            testMethods.isEmpty() -> emptyList<List<String>>().also { printNoTests(testApks) }
```

Comment:
I think `also` should be used if we want to perform some additional actions on caller object which is not true here. I would prefer simple block code. We can write it in one line if you want 
`testMethods.isEmpty() -> { printNoTests(testApks); emptyList() }`

Reply:
I agree 👍